cmd/stack_package.go

```diff
@@ -272,6 +264,14 @@ The packaging process builds the stack image, generates the "tar.gz" archive fil
 				cmdArgs = append(cmdArgs, options...)
 			}
 
+			labelPairs := CreateLabelPairs(labels)
```

Comment:
Could this change also be an issue in `build.go` when using build options?

Reply:
Think in `build.go` the `buildOptions` are already before labels are added